pkg/metrics/types.go

```diff
@@ -43,6 +43,13 @@ const (
 	// Control plane metrics
 	pluginManagerFailedToReconcileCounterDescription = "Number of times the plugin manager failed to reconcile the plugins"
 	lostEventsCounterDescription                     = "Number of events lost in control plane"
+
+	// Conntrack metrics
+	ConntrackPacketForwardDescription    = "Number of forward packets"
```

Comment:
forward or forwarded?

Reply:
I think we should use ‘forward’ since in this context we refer to ‘direction’ more than ‘action’. We also want to be consistent with the actual metrics implementation in BPF, which uses 'forward' and 'reply'. https://github.com/microsoft/retina/blob/f5fb767df6a2076222f02112ea5585ae2f699613/pkg/plugin/conntrack/_cprog/conntrack.c#L27

Reply:
Might be better, then, to say 'Number of packets marked "forward"'.

Reply:
I have updated this to `tx` and `rx` to avoid any confusion and remain consistent with the `ct_entry` naming convention.